assets/js/base/components/country-input/style.scss

```diff
@@ -0,0 +1,5 @@
+.wc-block-components-country-input,
+.wc-block-components-state-input {
+	margin-top: em($gap-large);
+	position: relative;
```

Comment:
Is `position: relative` needed? It looks like `.wc-block-components-select` already have it.